src/boot/store.js

```diff
@@ -163,6 +163,15 @@ const migrations: { [string]: (GlobalState) => GlobalState } = {
         typeof a.zulipVersion === 'string' ? new ZulipVersion(a.zulipVersion) : a.zulipVersion,
     })),
   }),
+
+  // Accounts.zulipFeatureLevel is `number | null`
+  '14': state => ({
+    ...state,
+    accounts: state.accounts.map(a => ({
+      ...a,
+      zulipFeatureLevel: a.zulipFeatureLevel !== undefined ? a.zulipFeatureLevel : null,
```

Comment:
As mentioned at https://github.com/zulip/zulip-mobile/pull/4079#discussion_r459144052 , this can just be:
```suggestion
      zulipFeatureLevel: null,
```